src/pages/utokyo_vpn/_help_cisco_secure_client.html

```diff
@@ -0,0 +1,12 @@
+<details>
+  <summary>ヘルプ：「VPNサービスが利用できないため、接続機能は使用できません」というエラーが表示される場合</summary>
+  以下の手順でCisco Secure Clientのバックグラウンドでの実行が許可されている状態にしてください．
+  
+  1. アプリケーションフォルダから「システム設定.app」を開いてください．
+  
+  2. 左の一覧から「一般」を選択し，さらに「ログイン項目と拡張機能」を選択してください．
```

Comment:
macOS 15 Sequoia、macOS 26 Tahoeのいずれのバージョンでも、「ログイン項目と**機能拡張**」という名称だと思います（もし手元では違う表記だったという場合は、ぜひ教えてください）。

```suggestion
  2. 左側の設定項目一覧から「一般」を選択し，さらに「ログイン項目と機能拡張」を選択してください．
```